README.md

```diff
@@ -128,6 +128,13 @@ To debug over Wi-Fi: http://blog.fossasia.org/android-app-debugging-over-wifi-fo
 Note :
 1. If you built your own hardware, change VendorID and/or ProductID in [CommunicationHandler.java](https://github.com/fossasia/pslab-android/blob/master/app/src/main/java/org/fossasia/pslab/communication/CommunicationHandler.java)
 
+### Permissions Required
+
+1. Record_Audio : It is required because in oscilloscope we are accepting inputs from the mic so we need to add this permission in order to contiune the proper functioning of this feature.You can find its implementation in [AudioJack.java](https://github.com/fossasia/pslab-android/blob/development/app/src/main/java/io/pslab/others/AudioJack.java).
```

Comment:
1. contiune --> continue
2. feature.You --> feature. You

Reply:
Keep it simple, don't explain for too long.
For oscilloscope to accept inputs from the phone microphone. You can find.....